zsh/zshrc.symlink

```diff
@@ -42,6 +42,7 @@ compinit
 prepend_path /usr/local/opt/grep/libexec/gnubin
 prepend_path /usr/local/sbin
 prepend_path $DOTFILES/bin
+prepend_path $HOME/.local/bin
```

Comment:
Can you remove this for the PR? It's not a directory that's set up by the install script and nothing from this setup is going in there.
```suggestion
```